search-books/src/main/java/nix/education/controller/BookController.java

```diff
@@ -0,0 +1,65 @@
+package nix.education.controller;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import nix.education.entity.Book;
+import nix.education.request.BookRequest;
+import nix.education.service.search.SearchService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BookController {
+
+    @Autowired
+    @Qualifier("solrSearchBookService")
+    private SearchService searchService;
+
+    @GetMapping("/books")
+    public List<Book> getBooks() {
+        return searchService.findAll();
+    }
+
+    @PostMapping("add")
```

Comment:
Let's add forward slash to the endpoint declaration for consistency

Reply:
Ok, agree